core/commands/generate_product_repositories_command.rb

```diff
@@ -439,9 +494,12 @@ def parse_mysql_rpm_repository(config)
   end
 
   STORED_KEYS = %i[repo repo_key platform platform_version product version].freeze
+  STORED_KEYS_DOCKER = %i[repo platform platform_version product version].freeze
```

Comment:
You can provide empty repo_key for the docker products and do not change this method.